models/profileDiffs.js

```diff
@@ -0,0 +1,27 @@
+const firestore = require("../utils/firestore");
+const profileDiffsModel = firestore.collection("profileDiffs");
+
+/**
+ * Fetches the pending profile diffs
+ * @return {Promise<profileDiffsModel|Array>}
+ */
+const fetchProfileDiffs = async () => {
```

Comment:
**Suggestion:** Add a unit test for this model function

Reply:
Can't write as pallab wrote a few functions in his PR after merging we will add that.